examples/routing_service/routing_service_file_adapter/cpp/FileStreamReader.cxx

```diff
@@ -0,0 +1,129 @@
+/*****************************************************************************/
+/*         (c) Copyright, Real-Time Innovations, All rights reserved.        */
+/*                                                                           */
+/*         Permission to modify and use for internal purposes granted.       */
+/* This software is provided "as is", without warranty, express or implied.  */
+/*                                                                           */
+/*****************************************************************************/
+
+#include <sstream>
+#include <thread>
+
+#include "FileStreamReader.hpp"
+
+using namespace rti::community::examples;
+using namespace dds::core::xtypes;
+
+const std::string FileStreamReader::INPUT_FILE_PROPERTY_NAME = "example.adapter.input_file";
+const std::string FileStreamReader::SAMPLE_PERIOD_PROPERTY_NAME = "example.adapter.sample_period_sec";
+
+void FileStreamReader::ProcessThread()
+{
+    stop_thread_ = false;
```

Comment:
this should be initialized in the constructor

Reply:
Done. Initialized in the initializer list.